### ISSUE TITLE
Using the State of the Ecosystem in a new Haskell Guide

### ISSUE THREAD

**Issue by reubenharry** [Gabriel439/post-rfc#194]
I'm developing a newcomer friendly Haskell Guide here (https://haskell-docs.netlify.app/), and I refer to your excellent State of the Ecosystem post a few times. It occurred to me that I could host the list inside the guide - either the whole thing, or just the packages in each section, without the prose (screenshot below to show what it would look like). Of course, I wouldn't do this without checking with you if it's ok (and if I did it, I'd credit you and link to this repo), but I thought there would be no harm in asking what you thought.

Thanks!

<img width="1258" alt="Screenshot 2023-02-11 at 3 28 00 PM" src="https://user-images.githubusercontent.com/7295980/218279795-e04f572a-166c-4b4c-b14f-672f49cc9c56.png">

**Comment by Gabriella439** [comment]
yeah, that's totally fine!  the only thing to be aware of are the terms of the [license](https://github.com/Gabriella439/post-rfc/blob/main/LICENSE.md)

in particular, you mainly just need to attribute the work
